resources/dicts/thoughts/alive/general.json

```
@@ -51,87 +49,1497 @@
+            "Goes to bed early so {PRONOUN/m_c/subject} can work more tomorrow"
+        ],
+        "main_trait_constraint": [
+            "responsible"
+        ]
+    },
+    {
+        "id": "responsible_to_app",
+        "thoughts": [
+            "Is reminding an apprentice to follow the code"
+        ],
+        "main_trait_constraint": [
+            "responsible"
+        ],
+        "random_status_constraint": [
+            "apprentice"
+            ]
+    },
+    {
+        "id": "rebellious_to_leadership",
+        "thoughts": [
+            "Had a lengthy debate with r_c",
+            "Rolled {PRONOUN/m_c/poss} eyes at a job r_c assigned {PRONOUN/m_c/object} to",
+            "Is being scolded for {PRONOUN/m_c/poss} rebellious actions"
+        ],
+        "main_trait_constraint": [
+            "rebellious"
+        ],
+        "random_status_constraint": [
+            "deputy",
+            "leader"
+            ]
+    },
+    {
+        "id": "responsible_kits",
+        "thoughts": [
+            "Is watching the kits to make sure none of them accidentally hurt each other",
+            "Is teaching r_c the importance of following the code"
+        ],
+        "main_trait_constraint": [
+            "responsible"
+        ],
+        "random_status_constraint":[
+            "kitten"
+        ],
+        "random_living_status":[
+            "living"
+        ]
+    },
+    {
+        "id": "righteous_warrior",
+        "thoughts": [
+            "Is praying to StarClan for guidance",
+            "Is reevaluating {PRONOUN/m_c/poss} morals",
+            "Hopes to help guide {PRONOUN/m_c/poss} Clanmates down the right path",
+            "Always cheers the loudest of any cat at naming ceremonies",
+            "Silently judges a Clanmate from afar",
+            "Wonders if everyone thinks their own thoughts, or if StarClan puts thoughts in their heads",
+            "Is judging outsiders for not following StarClan",
+            "Calms {PRONOUN/m_c/self} down by thinking how StarClan will see to things",
+            "Is teaching the kits about morals until they all fall asleep",
+            "Reported a Clanmate for breaking the code",
+            "Is appalled at a Clanmate blatantly lying" 
+        ],
+        "main_trait_constraint": [
+            "righteous"
+        ]
+    },
+    {
+        "id": "gen_shameless",
+        "thoughts": [
+            "Took an extra piece of fresh-kill from the pile",
+            "Is grooming intensely in clear view of everyone in camp",
+            "Is snoring at a ridiculous volume",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Is bragging about how great a hunter {PRONOUN/m_c/subject} are",
+            "Announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} heading to the dirtplace",
+            "Taunted a dog stuck behind a Twoleg fence",
+            "Is eating, chewing very loudly",
```

```suggestion
            "Is eating and chewing very loudly",
```
felt the phrasing was a little awkward